packages/dom-expressions/src/server.js

```diff
@@ -285,14 +287,15 @@ export function ssrStyle(value) {
 
 export function ssrElement(tag, props, children, needsId) {
   let result = `<${tag}${needsId ? ssrHydrationKey() : ""} `;
+  const skipChildren = VOID_ELEMENTS.test(tag);
```

Comment:
Early test so we can re-use for two cases:
- Checking for child-related properties
- Markup result (for self-closing)